drydock/templates/drydock/k8s/patches/pdb.yml

```
@@ -0,0 +1,84 @@
+{%- if MFE_HOST is defined and DRYDOCK_PDB_MINAVAILABLE_PERCENTAGE_MFE != 0 %}
```

```suggestion
{%- if MFE_HOST is defined and DRYDOCK_PDB_MINAVAILABLE_PERCENTAGE_MFE > 0 %}
```
I think it's better to use `greater than`. Please replace where it applies